game_eggs/steamcmd_servers/stationeers/egg-stationeers.json

```diff
@@ -4,18 +4,18 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2022-01-20T13:00:48-05:00",
+    "exported_at": "2022-06-18T13:00:00-00:00",
     "name": "Stationeers",
     "author": "sysadmin@whiteshield.ch",
-    "description": "Stationeers Server\r\n\r\ndefault.ini will be created once you start game. Command parameters override default.ini parameters\r\nDon't forget to change the RCON Password !!\r\n\r\nRemote Administrator:\r\nYou can send commands on web browser.\r\nLink : http:\/\/[dedicated server address]:[GamePort]\r\n\r\nDedicated Server Wiki ==> https:\/\/stationeers-wiki.com\/Dedicated_Server_Guide\r\n\r\nStopping the server don't save the game, for manual save use the Remote Administrator.",
+    "description": "Stationeers Server\r\n\r\nsetting.xml will be created once you start game. Command parameters override setting.xml parameters\r\n\r\nDedicated Server Wiki ==> https:\/\/stationeers-wiki.com\/Dedicated_Server_Guide\r\n\r\nStopping the server don't save the game, for manual save use the Console.",
     "features": [
         "steam_disk_space"
     ],
     "images": [
         "quay.io\/pterodactyl\/core:source"
```

Comment:
I believe it was called out to use the ghcr images such as `ghcr.io/parkervcp/games:source`